SerialPrograms/Source/PokemonSV/Programs/AutoStory/PokemonSV_AutoStoryTools.cpp

```diff
@@ -1065,6 +1090,76 @@ void move_cursor_towards_flypoint_and_go_there(
 }
 
 
+
+void change_date(
```

Comment:
This function looked familiar. Then I realized the ItemPrinterRNG does something different since it needs to keep track of when it commits.

Let's move this function into somewhere in NintendoSwitch/ since I feel like a lot of things can use it.